docs/install/openshift.md

```diff
@@ -0,0 +1,263 @@
+## Requirements
+
+Before proceeding, please ensure that you have an OpenShift cluster running K8s
+1.19+ (OpenShift 4.7+) and have Helm 3.5+ installed. In addition, you'll need to
+install the OpenShift CLI (`oc`) to authenticate to your cluster and create OpenShift
+resources.
+
+You'll also want to install the [latest version of Coder](https://github.com/coder/coder/releases/latest)
+locally in order to log in and manage templates.
+
+## Install Coder with OpenShift
+
+### 1. Authenticate to OpenShift and create a Coder project
+
+Run the following command to login to your OpenShift cluster:
+
+```console
+oc login <cluster-url>
+```
+
+This will configure your local `~/.kube/config` file with the cluster credentials
+needed when installing Coder via `helm`.
```

Comment:
Suggest deleting the helm part - oc does lots of things like get pods, describe nodes, etc.